docs/user-guide/kustomize.md

```diff
@@ -85,3 +85,22 @@ argocd app set <appName> --kustomize-version v3.5.4
 ## Build Environment
 
 Kustomize does not support parameters and therefore cannot support the standard [build environment](build-environment.md).
+
+## Kustomizing Helm charts
+
+It's possible to [render Helm charts with Kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/chart.md).
+Doing so requires that you pass the `--enable-helm` flag to the `kustomize build` command.
+This flag is not part of the Kustomize options within ArgoCD.
+If you would like to render Helm charts through Kustomize in an ArgoCD application, you have two options:
+You can either create a [custom plugin](https://argo-cd.readthedocs.io/en/stable/user-guide/config-management-plugins/), or modify the `argocd-cm` configmap, activating the `--enable-helm` flag globally for all Kustomize applications:
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+  namespace: argocd
+data:
+  kustomize.buildOptions: --load-restrictor LoadRestrictionsNone --enable-helm
```

Comment:
Is `--load-restrictor LoadRestrictionsNone` strictly necessary? If so, could you add a quick note explaining why?

Reply:
It is not, I just included it out of habit, but the example is clearer without it. It has been removed,  :+1: